dbt/models/qc/schema.yml

```diff
@@ -131,7 +135,9 @@ models:
           meta:
             table_name: asmt
             category: relationships
-            description: FMV should not increase more than $500k YoY
+            description: >
+              cur_year_fmv (ASMT TOTAL) should not increase more than $500k
+              over pri_year_fmv
```

Comment:
**suggestion (non-blocking):** Considering Brian's note about trying to make tests that are high signal, we might want to remove this one and others that look for % changes.

Reply:
Good thinking! I made this change in b57d1a1f538988d00c1c4a7c6a2e6e9ccb539920, and also got rid of the `qc.vw_incorrect_asmt_value` view and moved its remaining tests onto `asmt_all` and `pardat` (as we discussed offline).

One subtlety to this change is that `qc.vw_incorrect_asmt_value` was previously defining these tests on a rehydrated version of the missing-but-implied `asmt` table:

https://github.com/ccao-data/data-architecture/blob/e56518dd22b45d88d9803829541788856f62bbcf/aws-athena/views/qc-vw_incorrect_asmt_value.sql#L38-L51

In moving the remaining tests to `asmt_all`, I switched to use the correct filtering for `asmt_all` (filtering on `(procname, valclass, rolltype, deactivat)` instead of `(cur, deactivat)`) which resulted in slightly different failure counts for all tests except `iasworld_pardat_class_equals_luc:

| new test name | old failure count | new failure count | difference |
| --- | --- | --- | --- |
| iasworld_asmt_all_valasm1_non_ex_rr_gt_0 | 3497 | 3229 | -268 |
| iasworld_asmt_all_valasm2_impr_class_no_val | 957 | 352 | -605 |
| iasworld_asmt_all_valasm2_vact_class_has_val | 224 | 282 | +58 |
| iasworld_asmt_all_valasm3_non_ex_rr_gt_0 | 158 | 129 | -29 |
| iasworld_pardat_class_equals_luc | 2412 | 2412 | +0 |

Since most of the new failure counts are decreasing, and the only increasing one is going up by just a little bit, this seems OK to me; but let me know if you think my new filtering is incorrect, and we can troubleshoot.